lib/neeto-bugtrap-ruby/util/http.rb

```diff
@@ -44,7 +44,7 @@ def get(endpoint)
       end
 
       def post(endpoint, payload, headers = nil)
-        response = http_connection.post(endpoint, compress(payload.to_json), http_headers(headers))
+        response = http_connection.post(endpoint,  { payload: JSON.parse(payload.to_json).deep_symbolize_keys }.to_json, http_headers(headers))
```

Comment:
1. Do we need to `deep_symbolize_keys`?
2. Do we need to do to_json and then JSON.parse and then again to_json? I think, without any of this also it should work fine. Or at max, we can do it once.
```suggestion
        response = http_connection.post(endpoint,  { payload: JSON.parse(payload.to_json).deep_symbolize_keys }.to_json, http_headers(headers))
```

Reply:
Yes. I'll remove all those and just keep one `to_json`. Rest will be handled at the BugtrapWeb end.